src/tests/functional/plugin/conformance/test_runner/op_conformance_runner/include/read_ir_test/read_ir_compare_with_refs.hpp

```diff
@@ -3,13 +3,83 @@
 //
 
 #include "read_ir_test/read_ir.hpp"
+#include "functional_test_utils/crash_handler.hpp"
 
 namespace ov {
 namespace test {
 namespace subgraph {
 
-TEST_P(ReadIRTest, ReadIR) {
-    run();
+// TEST_P(ReadIRTest, ReadIR) {
+//     run();
+// }
+
+TEST_P(ReadIRTest, ImportExport) {
+    if (function == nullptr) {
+        GTEST_FAIL() << "Target model is empty!";
+    }
+
+    // in case of crash jump will be made and work will be continued
+    CommonTestUtils::CrashHandler crashHandler;
+
+    // place to jump in case of a crash
+    int jmpRes = 0;
+#ifdef _WIN32
+    jmpRes = setjmp(CommonTestUtils::env);
+#else
+    jmpRes = sigsetjmp(CommonTestUtils::env, 1);
+#endif
+    if (jmpRes == CommonTestUtils::JMP_STATUS::ok) {
+        crashHandler.StartTimer();
+        summary.setDeviceName(targetDevice);
+        try {
+            ov::CompiledModel model = core->compile_model(function, targetDevice, configuration);
```

Comment:
use comparator from `common_test_utils/graph_comparator.hpp` instead of checking in/out